deploy/cli/dapps.py

```diff
@@ -87,8 +89,20 @@ def report_data_to_markdown(df: pd.DataFrame) -> str:
     df.columns = [col.upper() for col in df.columns]
     df['GAS_USED_%'] = df['GAS_USED_%'].apply(lambda x: f"{x:.2f}")
 
+    def split_action(action):
```

Comment:
Looks like we need move everything connected reports to another file because now reports generates not only for dapps